azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -1,10 +1,30 @@
+from __future__ import annotations
 import sys
-from typing import TYPE_CHECKING, Union
+import types
+from typing import TYPE_CHECKING, Union, Callable
 
 if TYPE_CHECKING:
-    import pandas
+    import pandas as pd
     from azure.kusto.data._models import KustoResultTable, KustoStreamingResultTable
 
+try:
```

Comment:
I think we can avoid this.

You can

```python
from functools import lru_cache

@lru_cache
def default_dict() -> ...:
    return {...}

```


This will cache the result and you don't have to do the import hack.

Reply:
If you keep the import anyway, put it before the TYPE_CHECKING if